msal/application.py

```diff
@@ -240,8 +256,13 @@ def get_accounts(self, username=None):
         # but does NOT correlate them with saved RTs. It probably won't matter,
         # because in MSAL universe, there are always Accounts and RTs together.
         accounts = self.token_cache.find(
-            self.token_cache.CredentialType.ACCOUNT,
-            query={"environment": self.authority.instance})
+            self.token_cache.CredentialType.ACCOUNT, query={"environment": self.authority.instance})
```

Comment:
Nit pick: As controversial as it might be, Python coding style does recommend ["limit all lines to a maximum of 79 characters."](https://www.python.org/dev/peps/pep-0008/#maximum-line-length). Although in our code base we don't strictly enforce such rule and you can occasionally see 80+ characters per line, we don't usually combine existing short lines into one extra long line in a new PR. Please refer to the "reduce noise" topic below. :-)